### PR TITLE
DYN-7343: Pm search compatibility filters update

### PR DESCRIPTION
**Pull request by dnenov**
### Purpose

A few minor issues/alignments to push before 3.4. 

### UI Changes 

image

i9Pxc7kVEg

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB

### Release Notes

- `Compatible` and `Incompatible` filters are now exclusive @QilongTang @hwahlstrom 
- `*` to `x` conversion for the UI 
- minor color fix for alternating row colors for the `compatibility matrix`

### Reviewers

@zeusongit 
@QilongTang 

### FYIs

@reddyashish

### CODE REVIEW

**Review by zeusongit — APPROVED** [review]

File: src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml [hunk 1]
```diff
@@ -6480,7 +6480,7 @@
         <Setter Property="BorderBrush" Value="{StaticResource DarkMidGreyBrush}" />
         <Setter Property="ColumnWidth" Value="Auto" />
         <Setter Property="GridLinesVisibility" Value="Vertical" />
-        <Setter Property="VerticalGridLinesBrush" Value="{StaticResource DarkMidGreyBrush}" />
+        <Setter Property="VerticalGridLinesBrush" Value="#646464" />
```
Comment (QilongTang) [inline src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml:6483]: Hmm this looks like backward to me?

**Review by QilongTang — COMMENTED** [review]
One question

File: src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml (hunk 1 above)
Comment (dnenov) [reply src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml]: I updated this to match the design we have on Figma. It meant that I needed to add a new color to the swatch of colors we used so far. https://www.figma.com/design/MJh76m7bbafE3sjwLUTZhZ/Dynamo-package-manager-update?node-id=2397-31482&node-type=canvas&t=OCvkjGxUxDFeO1qW-0

I pushed a new commit making it a resource, which I should have done to begin with, apologies @QilongTang .

**Comment by dnenov** [comment]
Jenkins reports 1 test timing out, the test passes locally .. I don't see how it is connected with any of these changes.

image

**Comment by zeusongit** [comment]
> Jenkins reports 1 test timing out, the test passes locally .. I don't see how it is connected with any of these changes.
> 
> image

Merging since the test in unrelated.

And yes, I have seen that fail for other PRs too, is this related to this PR https://github.com/DynamoDS/Dynamo/pull/15583? fyi @reddyashish

**Comment by reddyashish** [comment]
The tests passed on that PR though. I will check that test again with those changes

**Pull request merged**